what

```diff
@@ -1,10 +1,12 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
 
 # Like "w", but finds all processes associated with a TTY (not just
 # those registered in wtmp), and reports all users that are running
 # anything.  In particular, unlike "w", this will shows things running
 # in detached screens/tmuxen.
 
+from __future__ import print_function
```

Comment:
Given the way we're using print, you actually don't need to import this. A print statement in Python 2 still works if the argument is parenthesized. And this way, it should even work with Python versions before 2.6.0a2.